modular_skyrat/modules/delam_emergency_stop/code/scram.dm

```diff
@@ -114,14 +129,9 @@
 
 	if(admin_disabled)
 		investigate_log("Delam SCRAM tried to activate but an admin disabled it", INVESTIGATE_ATMOS)
-		playsound(src, 'sound/misc/compiler-failure.ogg', 100, FALSE, 20, ignore_walls = TRUE, use_reverb = TRUE)
-		audible_message(span_danger("The [src] makes a series of sad beeps. Someone has corrupted its software!"))
-		return FALSE
-
-	if(world.time - SSticker.round_start_time > 30 MINUTES && trigger_reason != DIVINE_INTERVENTION)
-		playsound(src, 'sound/misc/compiler-failure.ogg', 100, FALSE, 20, ignore_walls = TRUE, use_reverb = TRUE)
-		audible_message(span_danger("The [src] makes a series of sad beeps. The internal charge only lasts about 30 minutes... what a feat of engineering!"))
-		stack_trace("Delam SCRAM was triggered with an invalid time or trigger reason!")
+		playsound(src, 'sound/misc/compiler-failure.ogg', 100, FALSE, 15, ignore_walls = TRUE, use_reverb = TRUE, falloff_distance = MACHINE_SOUND_FALLOFF_DISTANCE)
```

Comment:
Just because I can't remember it off the top of my head, is the `15` here supposed to be for the range? If so, I think that one should be a define as well.

Reply:
Yeah, it is.